documentation/src/docs/pravega-write-read-methods.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Methods for Writing and Reading Pravega Streams
+---
+
+import { IfHaveFeature, IfMissingFeature, SDPVersion } from 'nautilus-docs';
+
+## Overview
+
+This table describes the methods that can be used to write and read data in Pravega streams. It is intended to be a comprehensive list which includes open-source, experimental, and commercial products that integrate with Pravega.
+
+Method                                                                                          | Write | Transactional Write | Read | Special Features | Notes
+------------------------------------------------------------------------------------------------|-------|---------------------|------|------------------|------------------------------------------------------
+[Pravega Sensor Collector](#pravega-sensor-collector)                                           | Yes   | Yes                 | No   | EO               |
```

Comment:
There are links not working in this table. We also may need to consider a couple of things bout this table: i) should we order the list in terms of popularity and/or client completeness? For those really experimental connectors or under development, should we note that in the last column?

Reply:
Fixed table links and entry descriptions. Still open questions about columns/completeness.